speech/wave_proc.m

```diff
@@ -31,7 +31,7 @@
 %% pitch
 if params2track(1)
     
-    [y_pitch, pitch_taxis] = get_sig_pitch(y, fs, sigproc_params.pitchlimits, [], [], [], sigproc_params);
+    [y_pitch, pitch_taxis] = get_sig_pitch(y, fs, sigproc_params, [], [], []);
```

Comment:
Can get rid of []s

Reply:
Fixed in 34bac2e477d9401a3b4bbabafee7246a3a1856d9